src/Lobby.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Socket } from "socket.io-client";
+
+import LobbyForm from "LobbyForm";
+import React, { ReactElement } from "react";
+import { css, StyleSheet } from "aphrodite";
+import logo from "./static/logo.png";
+
+type Props = Readonly<{
+  socket: Socket;
+}>;
+
+function Lobby({ socket }: Props): ReactElement | null {
+  return (
+    <div>
+      <div className={css(styles.top)}>
+        <img className={css(styles.logo)} src={logo} alt="logo" />
+      </div>
+      <div className={css(styles.words)}>
+        <h1 className={css(styles.dicey)}>DICEY</h1>
+        <h2 className={css(styles.iRoll)}>This is how I roll</h2>
+      </div>
+      <div className={css(styles.main)}>
+        <LobbyForm socket={socket} />
+      </div>
+    </div>
+  );
+}
+
+const styles = StyleSheet.create({
+  main: {
+    display: "flex",
+    flexDirection: "column",
+    backgroundColor: "#1D1E33",
+  },
+  words: {
+    textAlign: "center",
+  },
+  top: {
+    flexDirection: "column",
+    textAlign: "center",
+    justifyContent: "center",
```

Comment:
does flexDirection or justifyContent do anything here? it's just a div wrapping an img right?